### DISCUSSION TITLE
Releases vs. tags

### DISCUSSION THREAD

**Discussion by Zebradil** [GoogleCloudPlatform/prometheus-engine#656] [Q&A] [answered]
At the moment, the latest release is [v0.7.4](https://github.com/GoogleCloudPlatform/prometheus-engine/releases/tag/v0.7.4) but the latest tag is [v0.8.0](https://github.com/GoogleCloudPlatform/prometheus-engine/releases/tag/v0.8.0). Which version should I use?

**Comment by pintohutch** [comment, marked as answer]
Hey @Zebradil - excellent question!

The answer to your question is: use the latest non-release candidate (non-`rc.C` suffix) tag.

The longer answer: we hold off on promoting a "Tag" to a "Release" until that version is readily available as an upgrade target in the GKE Regular [release channel](https://cloud.google.com/kubernetes-engine/docs/release-schedule#schedule-for-release-channels).

The reason being: when people ask how they can upgrade to the latest version, we can provide a recommended path forward (i.e. `gcloud container clusters upgrade`) for those on GKE that will capture a majority of users.

cc @lyanco

**Reply by Zebradil** [reply]
Great answer, thanks! :-)
